src/SFA.DAS.EmployerIncentives.Data.UnitTests/TestHelpers/SqlDatabaseForTest.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.IO;
+
+namespace SFA.DAS.EmployerIncentives.Data.UnitTests.TestHelpers
+{
+    public class SqlDatabaseForTest : IDisposable
```

Comment:
It's implicit from the namespace that it's for test so maybe just call it SqlDatabase.

Reply:
will do after i replace the original